tests/test_one_clicks.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+
+from dolib.client import AsyncClient, Client
+
+
+@pytest.mark.vcr
+@pytest.mark.block_network()
+def test_crud_one_clicks(client: Client) -> None:
+
+    # list one click app
+    apps = client.one_clicks.all()
+    apps_count = len(apps)
+    assert apps_count > 0
+
+    # empty filtered app
+    empty_type_apps = client.one_clicks.filter()
+    assert len(apps) == len(empty_type_apps)
+
+    # filter one click app
+    filtered_apps = client.one_clicks.filter(app_type="droplet")
+    assert isinstance(filtered_apps, list)
+    for app in filtered_apps:
```

Comment:
I would check `assert len(filtered_apps) == 1` before the `for` to make sure that assert inside the `for` block is called.